x-pack/plugins/security_solution/cypress/e2e/entity_analytics/entity_analytics_management_page.cy.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  PAGE_TITLE,
+  HOST_RISK_PREVIEW_TABLE,
+  HOST_RISK_PREVIEW_TABLE_ROWS,
+  USER_RISK_PREVIEW_TABLE,
+  USER_RISK_PREVIEW_TABLE_ROWS,
+  RISK_PREVIEW_ERROR,
+  RISK_PREVIEW_ERROR_BUTTON,
+} from '../../screens/entity_analytics_management';
+
+import { login, visit, visitWithoutDateRange } from '../../tasks/login';
+import { cleanKibana } from '../../tasks/common';
+import { ENTITY_ANALYTICS_MANAGEMENT_URL, ALERTS_URL } from '../../urls/navigation';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+import { getNewRule } from '../../objects/rule';
+import { createRule } from '../../tasks/api_calls/rules';
+import { updateDateRangeInLocalDatePickers } from '../../tasks/date_picker';
+import { fillLocalSearchBar, submitLocalSearch } from '../../tasks/search_bar';
+
+describe('Entity analytics management page', () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('all_users');
+  });
+
+  beforeEach(() => {
+    login();
+    visitWithoutDateRange(ALERTS_URL);
+    createRule(getNewRule({ query: 'user.name:* or host.name:*', risk_score: 70 }));
+    visit(ENTITY_ANALYTICS_MANAGEMENT_URL);
+  });
+
+  after(() => {
+    esArchiverUnload('all_users');
+  });
+
+  it('renders page as expected', () => {
```

Comment:
Hey @rylnd!! With the improvements @patrykkopycinski and @szwarckonrad introduced lately, Cypress tests are pretty fast on CI. So after the introduced improvements and for this specific case (each case may be different), I'm ok with any of the proposed solutions. No objection to consolidating or keeping it as it is to improve readability. In case we move to the consolidation scenario, we just need to make sure that the names of the tests reflect what we are testing and it is pretty clear which is the scope of it.